AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/RequestTask.java

```diff
@@ -350,6 +360,16 @@ protected void onPostExecute(String result) {
         }
     }
 
+    protected void recheckCurrentCell() {
+        if (mAimsicdService == null) {
+            log.warn("AIMSICD service not set, unable to recheck current cell post-OCID update.");
+            return;
+        }
```

Comment:
Right now this will always evaluate to true until I know how to give the task access to the service.